Directory.Build.props

```diff
@@ -0,0 +1,28 @@
+<Project>
+  <!-- These properties will be shared for all projects -->
+
+  <PropertyGroup>
+    <Product>VirtoCommerce</Product>
+    <Copyright>Copyright © VirtoCommerce 2011-2020</Copyright>
+    <Authors>VirtoCommerce</Authors>
+  </PropertyGroup>
+   <PropertyGroup>
+    <VersionPrefix>5.0.0</VersionPrefix>
+    <VersionSuffix></VersionSuffix>
+    <VersionSuffix Condition=" '$(VersionSuffix)' != '' AND '$(BuildNumber)' != '' ">$(VersionSuffix)-$(BuildNumber)</VersionSuffix>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(MSBuildProjectName.Contains(Tests))' == false">
```

Comment:
Looking to these settings you will publish nugget for Storefront project web app, it does not make sense. Use 
```<IsPackable>False</IsPackable>``` XML tag for search project file instead to control which projects are packable.